config_converter/config_mapper/config_mapper.py

```diff
@@ -85,19 +87,22 @@ def extract_root_dirs(config_obj: config_pb2.Directive) -> dict:
         plugin_name = plugin_prefix_map[d.name] + plugin_type
         if plugin_name not in _SUPPORTED_PLUGINS:
             stats['entities_unrecognized'] += 1
-            stats['attributes_unrecognized'] += get_num_attrs(d)
+            stats['attributes_unrecognized'] +=\
+                    _get_aggregated_num_attributes(d)
             print(f'We do not know plugin {plugin_name}')
         else:
             plugin_dir = dir_name_map[d.name]
             if plugin_dir not in logs_module:
                 logs_module[plugin_dir] = []
-            old_attrs = stats['attributes_recognized']
+            current_attribute_count = stats['attributes_recognized']
             logs_module[plugin_dir].append(
                 _convert_plugin(d, plugin_name, stats))
```

Comment:
```suggestion
                _convert_plugin(d, plugin_name, stats)) // stats are updated.
```
Without reading into the implementation, the reader doesn't know the `stats` dictionary are modified - and I think it's a bit confusing. So I think it may be good to add a comment to inform the reader.

Reply:
Good point - will change!